transport/src/main/java/io/netty/channel/group/VoidChannelGroupFuture.java

```diff
@@ -27,12 +28,17 @@
 final class VoidChannelGroupFuture implements ChannelGroupFuture {
 
     private static final Iterator<ChannelFuture> EMPTY = Collections.<ChannelFuture>emptyList().iterator();
-    private final ChannelGroup group;
+    private final DefaultChannelGroup group;
```

Comment:
The `ChannelGroup` -> `DefaultChannelGroup` is needed?

Reply:
yes as we access `group.executor`